yaml/argon2.yaml

```
@@ -1,5 +1,6 @@
 # SPDX-License-Identifier: CC0-1.0
 id: 'argon2'
+oid: 1.3.6.1.4.1.50370.1
```

```suggestion
oid: 
```
It look like not an OID for Argon2